.circleci/config.yml

```diff
@@ -20,6 +20,9 @@ commands:
             tar --strip-components=1 -C $GOPATH -xzf $GOPATH/go.tar.gz
             rm -rf $GOPATH/go.tar.gz
       - checkout
+      - restore_cache:
+          keys:
+            - 'v1-pkg-cache-{{ checksum "go.sum" }}-{{ .Environment.GOVERSION }}'
```

Comment:
Do we need to also cache by os/arch?

Reply:
Hm, since this is a source cache, I would say no. But if were caching the object cache (which we know a bit about, thanks to Testground adventures cc @nonsense) then yes.

Reply:
Worth reading, if you're interested: https://golang.org/cmd/go/#hdr-Build_and_test_caching